packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts

```diff
@@ -822,9 +823,9 @@ export function canAddSso(
   // Can not add sso if feature flag is not enabled
   if (!isAadManifestEnabled()) {
     const e = new SystemError(
+      SolutionSource,
       SolutionError.NeedEnableFeatureFlag,
-      getLocalizedString("core.addSso.needEnableFeatureFlag"),
-      SolutionSource
+      getLocalizedString("core.addSso.needEnableFeatureFlag")
     );
```

Comment:
tip: can we hide add sso command palette in package.json file? So that we do not need this check any more

Reply:
Actually it is already disabled in the command palette. Here it is only to ensure user we won't enable this feature by mistake.